File: corellia/src/test/java/org/example/product/corellia/test/CorelliaModuleTest.java
package org.example.product.corellia.test;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.example.product.corellia.fixtures.CorelliaModuleFixture;
import org.junit.jupiter.api.Test;

public class CorelliaModuleTest {

    @Test
    void testModule() {
        assertEquals(3, new CorelliaModuleFixture().threeModules().size());
    }
}

File: corellia/src/test/kotlin/org/example/product/corellia/test/CorelliaModuleTest.kt
package org.example.product.corellia.test

import org.example.product.corellia.fixtures.CorelliaModuleFixture
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class CorelliaModuleTest {
    @Test
    fun testModule() {
        Assertions.assertEquals(3, CorelliaModuleFixture().threeModules().size)
    }
}
